### ISSUE TITLE
What's the correct way to use these form fields? `v-model` seems to have mixed support

### ISSUE THREAD

**Issue by mikemike** [HulkApps/polaris-vue#55]
As the title suggests I'm struggling to use this package to create a working form. The form itself displays as expected, but any use of `v-model` on anything other than a textfield (so a select, for instance) doesn't work. 

# Expected behaviour
Add v-model to any form element (`PSelect`, for example) and it will be accessible within the component.

# Actual behaviour
Adding `v-model` to a `PSelect` does nothing. The data isn't available in the component.

# Example code
```
<template>
    <PFormLayout>
        <PHeading>Products</PHeading>
        
        <PTextField
            label="How many products?"
            type="number"
            helpText="This is the number of products we'll generate. There's a maximum of 100 at a time."
            value="20"
            v-model="form.num"
        />

        <PSelect
            label="Variations?"
            :options="['No', 'Yes', 'Mixed']"
            v-model="form.has_variations"
        />

        <PButton :primary="true" :loading="isBusy" @click="run()">Go</PButton>
    </PFormLayout>
</template>

<script>
    export default {

        data() {
            return {
                form: {
                    num: 20,
                    has_variations: '',
                },
                isBusy: false,
                hasError: false,
                errors: '',
            }
        },

        methods: {
            // Generate the products
            run() {
                this.isBusy = true;
                console.log(this.form);
            },
        }
    }
</script>

```

In this example I'd expect `this.form` to contain the contents of the form, eg the number entered into the `form.num` field and the item selected. I *can access* the textfield value, but I can't access the `PSelect`.

I imagine pretty much everyone uses forms who uses this package so I imagine I'm approaching this the wrong way. Am I expected to perhaps name or ID each field and then look up the value on a more manual basis?

**Comment by uttam-hulkapps** [comment]
@mikemike 

Thank you for pointing out the issue here. We've resolved this issue, it will be released with our next release. It may take us day or two to release next version, meanwhile you can use @change event to update the data value for PSelect Component.

Thank you.

**Comment by uttam-hulkapps** [comment]
@mikemike 
This issue has been fixed in v2.3.2, kindly please update the library.
